lib/ansible/modules/system/user.py

```diff
@@ -726,6 +736,33 @@ def chown_homedir(self, uid, gid, path):
             e = get_exception()
             self.module.exit_json(failed=True, msg="%s" % e)
 
+    def parse_defaults(self):
+        try:
+            f = open('/etc/login.defs', 'r')
+            for line in f:
+                if 'UMASK' in line and '#' not in line.split()[0]:
+                    maskvalue=line.split()[1]
+                    self.newumask = int(maskvalue, 8)
+            f.close()
+        except:
```

Comment:
The wide exception catching is a bit weird, as it would mask all kind of errors. Could it be restricted more ?